_activities/toolsandpackaging.md

```diff
@@ -11,26 +11,27 @@ redirect_from:
 
 ## Software Developer Tools
 
-The Software Developer Tools Working Group aims to help the HEP community in developing the latest, modern feature-rich projects
+The Software Developer Tools Activity Area aims to help the HEP community in developing the latest, modern feature-rich projects
 with the aid of software tools such as code editors, static/dynamic code analysers, compilers, debuggers, performance analysers and so on.
 The group plans to accomplish this by mainly serving as a forum where developers can share their experiences, discuss new/existing tools they utilise,
 and learn from each other. To that point, contributions from the entire community are strongly encouraged.
 
 ## Packaging
 
-The group has a particular focus also on solutions for building and deploying software stacks, through the evolution of the previous *HSF Packaging Working Group*
+The group has a particular focus also on solutions for building and deploying software stacks, through the evolution of the previous *HSF Packaging Working Group*.
```

Comment:
Maybe we should talk about `this area` instead of a group

Reply:
I left "group" in a few places as in "group of people", but happy with a change. I have no strong feeling here.